services/registry/models/token.js

```diff
@@ -53,6 +52,10 @@ module.exports = class Token {
   }
 
   static async lookupUser(value) {
+    if (!value || value.indexOf('ent_v1_') !== 0) {
```

Comment:
Sanity check added here.

Reply:
🎉 This is a great idea! (A tiny nit, but could we switch from `indexOf` to `startsWith`?)

Reply:
Sure, updated